src/views/event.hbs

```diff
@@ -0,0 +1,25 @@
+<div class="">
+  <div class="">
+    <div class="">
+      <h1>{{this.events.eventTitle}}</h1>
+      <p>{{this.events.eventDescription}}</p>
+    </div>
+    <div class="">
+      <a href="#">RSVP</a>
+      <span>{{this.events.eventDate}}</span>
```

Comment:
this.eventS.EVENTdate

this should be changed to 'this.event.eventDate'
and then fix it in the backend